src/server/routes/image-api.js

```diff
@@ -67,6 +67,14 @@ router.get('/image/:galleryId', (req, res) => {
   });
 });
 
+router.get('/image/:id/details', (req,res) => {
```

Comment:
wait, the endpoint name is ´/details´ but you return an actual image? How is this any different from `/image/:id/fullSize` ?